### ISSUE TITLE
Basic Layout

### ISSUE THREAD

**Issue by m-henderson** [m-henderson/tenantbuddy#1]
I like the nav on the left and the bar on top. I think that would be a good start. Here is an example of a site that looks simple (highlighted in red):

<img width="1676" alt="Screenshot 2024-03-10 at 12 21 40 PM" src="https://github.com/m-henderson/tenantbuddy/assets/16070192/a6df0123-73ea-4442-bdae-5918e8663a55">

**Comment by m-henderson** [comment]
Might need to use media queries for different devices sizes, etc. Im going to use a fixed height for now.

**Comment by momaqbol** [comment]
@m-henderson can you send screenshots of the same site you had in the issue but for different breakpoints? Maybe we can take inspirations from that!

**Comment by m-henderson** [comment]
@Enjoy2Live would it show the break points here in the bottom right if there were any? 
<img width="1680" alt="Screenshot 2024-03-11 at 9 16 35 PM" src="https://github.com/m-henderson/tenantbuddy/assets/16070192/20d9ec52-29ff-492c-8b82-8efb1f525f1b">

**Comment by momaqbol** [comment]
@m-henderson 
No, this one is for adding breakpoints on DOM elements for debugging purposes.
Example: enter the debugger when router-outlet changes in DOM.

I wanted to see how their layout looks like for different screen sizes so can you include screenshots on 430px, 640px, 768px, 1024px screen widths here

**Comment by m-henderson** [comment]
@Enjoy2Live I think that is a great idea. is this what you are talking about?: 

### 430px
<img width="706" alt="image" src="https://github.com/m-henderson/tenantbuddy/assets/16070192/7fb2e027-06a1-4a7e-acb0-c6c3d90cebc2">
### 640px
<img width="745" alt="image" src="https://github.com/m-henderson/tenantbuddy/assets/16070192/7f1a9a76-f8f9-4fbc-b2ef-ea9f25bf0d4e">
### 768px
<img width="894" alt="image" src="https://github.com/m-henderson/tenantbuddy/assets/16070192/1bf8ab62-a55a-41d0-89cc-81a7a390fa81">
### 1024px
<img width="1255" alt="image" src="https://github.com/m-henderson/tenantbuddy/assets/16070192/cbb13a7d-325d-4e01-b039-9ecc9bdc0dc5">

**Comment by momaqbol** [comment]
Yup that's exactly what I wanted. now I have an idea of they handle layout on different breakpoints!
